test/test_torch.py

```diff
@@ -2683,6 +2683,17 @@ def test_qtensor(self):
         # This calls _th_fill_
         # qr[0] = 8 # float asignment
         # self.assertEqual(qr.item(), 8)
+        # we can also print a qtensor
+        self.assertEqual(str(qr),
+                         "tensor(is_quantized=True, qscheme=per_tensor_affine,\n" +
```

Comment:
why do you need to print is_quantized instead of just the dtype?  The dtype implies it's quantized, correct?

Reply:
I'd also move all of the metadata (e.g. scale, zero_point), to the end of the print, like it is currently with non-quantized tensors.

Reply:
The way we check for whether a QTensor is quantized internally is checking whether the underlying TensorTypeId is QuantizedCPUTensorId rather than whether the data type is QInt, so I feel it might be better to print is_quantized.

Reply:
probably we don't need that, qscheme already tells it's a QTensor